### ISSUE TITLE
Have one softmax_logits function

### ISSUE THREAD

**Issue by henrikfo** [aidotse/LeakPro#112]
# Feature
## Desired Behavior / Functionality
Instead of having multiple instances of the function softmax_logits in different files, have one single one in a utils.py file or similar

## What Needs to Be Done
Move the softmax function into a shared file. Remove all other instances of it and import it instead.

## How Can It Be Tested
-
